src/frontend/apps/e2e/__tests__/app-desk/footer.spec.ts

```diff
@@ -87,4 +87,18 @@ test.describe('Footer', () => {
       await expect(page).toHaveURL(url);
     });
   }
+
+  test('check if the app version is visible', async ({ page }) => {
+    await page.waitForResponse(
+      (response) =>
+        response.url().includes('/config/') && response.status() === 200,
+    );
+
+    const footer = page.locator('footer').first();
+    await expect(
+      footer.getByText(
+        'Version: NA • Unless otherwise stated, all content on this site is under',
+      ),
+    ).toBeVisible();
+  });
```

Comment:
Ah ok, I thought you will get it from a variable from response.
Like that it is fine as well, but you don't need this part so:

```suggestion
  test('check if the app version is visible', async ({ page }) => {
   const footer = page.locator('footer').first();
    await expect(
      footer.getByText(
        'Version: NA • Unless otherwise stated, all content on this site is under',
      ),
    ).toBeVisible();
  });
```